taxonomy-editor-frontend/src/pages/search/results.jsx

```diff
@@ -67,141 +65,161 @@ const SearchResults = () => {
+                                        <IconButton onClick={event => handleClick(event, node) } aria-label="edit">
+                                            <EditIcon color="primary"/>
+                                        </IconButton>
+                                    </TableCell>
+                                </TableRow>
+                            ))}
+                        </TableBody>
+                    </Table>
+                </TableContainer>
+                {/* Dialog box for adding nodes */}
+                <Dialog open={openAddDialog} onClose={handleCloseAddDialog}>
+                    <DialogTitle>Add a node</DialogTitle>
+                    <DialogContent>
+                    <Stack direction="row" alignItems="center" sx={{mt: 1, ml: 2, mr: 2, mb: 2}}>
+                        <Typography>Type of node</Typography>
+                        <FormControl sx={{ml: 6.5}}>
+                            <InputLabel>Type</InputLabel>
+                            <Select
+                                native
+                                label="Type"
+                                value={nodeType}
+                                onChange={(e) => {
+                                    setNodeType(e.target.value);
+                                }}
                             >
-                                <TableCell align="left" component="td" scope="row">
-                                    <Typography variant="subtitle1">
-                                        {node}
-                                    </Typography>
-                                </TableCell>
-                                <TableCell align="left" component="td" scope="row">
-                                    <IconButton onClick={event => handleClick(event, node) } aria-label="edit">
-                                        <EditIcon color="primary"/>
-                                    </IconButton>
-                                </TableCell>
-                            </TableRow>
-                        ))}
-                    </TableBody>
-                </Table>
-            </TableContainer>
-            {/* Dialog box for adding nodes */}
-            <Dialog open={openAddDialog} onClose={handleCloseAddDialog}>
-                <DialogTitle>Add a node</DialogTitle>
-                <DialogContent>
-                <Stack direction="row" alignItems="center" sx={{mt: 1, ml: 2, mr: 2, mb: 2}}>
-                    <Typography>Type of node</Typography>
-                    <FormControl sx={{ml: 6.5}}>
-                        <InputLabel>Type</InputLabel>
-                        <Select
-                            native
-                            label="Type"
-                            value={nodeType}
-                            onChange={(e) => {
-                                setNodeType(e.target.value);
+                                {/* TODO: Add support for synonyms and stopwords */}
```

Comment:
please create an issue or something to have it in mind somewhere

Reply:
I'll create it ASAP.